src/main/java/org/infernalstudios/infernalexp/InfernalExpansion.java

```diff
@@ -73,6 +74,7 @@
 import org.infernalstudios.infernalexp.network.IENetworkHandler;
 import org.infernalstudios.infernalexp.util.CompatibilityQuark;
 import org.infernalstudios.infernalexp.util.ModSpawnEggItem;
+import org.infernalstudios.infernalexp.util.NoiseChunkGeneratorUtil;
```

Comment:
Why did you add imports but not change anything else in this file?

Reply:
ah, this is an artifact of testing, will remove